SimplifiedTests/jwk_private

```diff
@@ -0,0 +1,25 @@
+MIIEpAIBAAKCAQEA7GcBZtUVccXneZdDu5KjhxUOo6cxqM718Uaiw6kZns+AGz/2
```

Comment:
is this and the json file above the actual key from Feedbooks or just a random test key? sorry if it's an obvious question, just want to make sure.

Reply:
it actually was, I was testing to check if it matches the key in the secrets file. replaced with a random one.

Reply:
thanks -- can you also make sure to squash the 2 commits so at least it's a bit more hidden?

Reply:
sure, squashed